playbilling/src/androidTest/java/com/google/androidbrowserhelper/playbilling/provider/WrapperActivity.java

```diff
@@ -0,0 +1,67 @@
+package com.google.androidbrowserhelper.playbilling.provider;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+import android.util.Log;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+/**
+ * An Activity that launches another Activity with {@link #startActivityForResult}.
+ *
+ * Originally I'd attempted to have this Activity launched by an ActivityTestRule, but that seemed
+ * unable to handle an Activity that immediately launched another Activity.
+ *
+ * I agree that having statics all over the place is ugly, but it's test code and it works.
```

Comment:
Since it's test code, should the name of the class reflect that somehow?

Reply:
It's in the androidTest build target - it seems a bit odd to have everything in there include the word Test in its name.